config/install/islandora.settings.yml

```diff
@@ -2,3 +2,4 @@ broker_url: 'tcp://localhost:61613'
 jwt_expiry: '+2 hour'
 delete_media_and_files: TRUE
 gemini_pseudo_bundles: []
+allow_header_links: TRUE
```

Comment:
An update hook should be implemented to create/populate this new configuration value when updating existing installations.